packages/cli-repl/src/cli-repl.ts

```diff
@@ -287,7 +287,22 @@ export class CliRepl implements MongoshIOProvider {
       delete driverOptions.autoEncryption;
     }
 
-    const initialServiceProvider = await this.connect(driverUri, driverOptions);
+    await this.prepareOIDCOptions(driverOptions);
```

Comment:
For me, it would read more clearly if the function returned the value, rather than updated `driverOptions` under the hood. Is there some best practice depending on the use case?

Reply:
(updated, although github isn't showing the update right now...)